tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -1984,6 +1984,27 @@ exports[`interface.test.w 1`] = `
    |   ^^^^^^^^^ Properties are not supported in interfaces
 
 
+error: Access modifiers are not allowed in interfaces
+   --> ../../../examples/tests/invalid/interface.test.w:36:3
+   |
+36 |   pub inflight foo(body: Json): Json;  
+   |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Access modifiers are not allowed in interfaces
+
+
+error: Unexpected 'identifier'
```

Comment:
This seems like an unwanted side effect. I don't think we want this error.

Reply:
That error on line 1994 existed before this PR's changes when running the same Wing code, so this error looks valid AFAIK. eg:
pub interface IBedrock {
  pub inflight invokeModel(body: Json): Json;  
}
**_pre-PR, produces_**:
Error at debug.main.w:83:3 | Properties are not supported in interfaces
Error at debug.main.w:83:28 | **Unexpected 'identifier'**
Error at debug.main.w:83:34 | Unexpected 'json_container_type'
**_post-PR, produces_**:
Error at debug.main.w:83:3 | Access modifiers are not allowed in interfaces
Error at debug.main.w:83:28 | **Unexpected 'identifier'**
Error at debug.main.w:83:34 | Unexpected 'json_container_type'

Reply:
I think you are correct @autow0rk.
To avoid adding these unrelated error messages, I suggest we change the invalid example on examples/tests/invalid/interface.test.w to:
```wing
interface IPointy {
    pub method2(): str;
}
```